python/opae.admin/opae/admin/fpga.py

```diff
@@ -415,13 +415,7 @@ def secure_dev(self):
         spi = f.spi_bus
         if spi:
             ifpga_sec = spi.find_one(
```

Comment:
optional nit: change the variable ifpga_sec to fpga_sec?

Reply:
fixed